### PR TITLE
Remote cluster fixes

### PR DESCRIPTION
**Pull request by orianac**
This PR shifts session/permissions-handling such that each worker gets the adequate permissions to read and write. These changes don't break the local cluster behavior (you can now just run either locally or on a remote- aka potentially bigger- cluster).

Also a change at L105 in `landsat_preprocess.py` to scale the reflectance values appropriately.

### CODE REVIEW

File: carbonplan_trace/v1/inference.py [hunk 1]
```diff
@@ -118,14 +122,17 @@ def convert_to_lat_lon(df, utm_zone_number, utm_zone_letter):
     return utm.to_latlon(df['x'], df['y'], int(utm_zone_number), utm_zone_letter)
 
 
-def load_xgb_model(model_path, local_folder='./'):
+def load_xgb_model(model_path, fs, local_folder='./'):
```
Comment (tcchiao) [inline carbonplan_trace/v1/inference.py:125]: remove `local_folder` argument since it's not used

File: carbonplan_trace/v1/inference.py [hunk 2]
```diff
@@ -160,24 +167,19 @@ def predict(
...
     season='JJA',
 ):
-
+    core_session = boto3.Session(
+        aws_access_key_id=access_key_id,
+        aws_secret_access_key=secret_access_key,
+        region_name='us-west-2',
+    )
+    aws_session = AWSSession(core_session, requester_pays=True)
+    fs = S3FileSystem(key=access_key_id, secret=secret_access_key, requester_pays=True)
```
Comment (tcchiao) [inline carbonplan_trace/v1/inference.py:176]: interesting that we need to move the session definition out! just curious if you know why this is?

File: carbonplan_trace/v1/landsat_preprocess.py [hunk 3]
```diff
@@ -102,7 +102,7 @@ def grab_ds(item, bands_of_interest, cog_mask, utm_zone, utm_letter):
     ds = ds.assign_coords({'band': bands_of_interest})
     ds = ds.where(ds != 0)
     ds = ds.where(cog_mask < 2)
-    ds = ds.reflectance.to_dataset(dim='band')
+    ds = ds.reflectance.to_dataset(dim='band') * 0.0000275 - 0.2
```
Comment (tcchiao) [inline carbonplan_trace/v1/landsat_preprocess.py:105]: can you add a comment as to why do this?

**Review by tcchiao — APPROVED** [review]

File: carbonplan_trace/v1/inference.py (hunk 2 above)
Comment (orianac) [reply carbonplan_trace/v1/inference.py:176]: short answer is i'm not sure. i _think_ what might have contributed was that i was calling `with rio.Env(aws_session)` but then re-assigning `aws_session` on the next line. we need to ensure that the session is created on the worker but with the permissions associated with the scheduler. i'm also wondering whether the `with dask.config` call might trigger some shift from scheduler to worker permissions and we want the session to be immune to any other things that call might cause.

File: carbonplan_trace/v1/inference.py (hunk 1 above)
Comment (orianac) [reply carbonplan_trace/v1/inference.py:125]: great

File: carbonplan_trace/v1/landsat_preprocess.py (hunk 3 above)
Comment (orianac) [reply carbonplan_trace/v1/landsat_preprocess.py:105]: done

**Pull request merged**